C/Clipper/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "Clipper"
+version = v"1.0.0"
+
+# Collection of sources required to build Clipper
+sources = [
+    GitSource("https://github.com/SimonDanisch/ClipperBuilder.git",
+              "9ea1878235d518d5e14bce35a94924a34bab8b68"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ClipperBuilder/
+mkdir "${libdir}"
+${CXX} -fPIC -std=c++11 -shared -o "${libdir}/libcclipper.${dlext}" clipper.cpp cclipper.cpp
+"""
```

Comment:
The license that should be installed is http://www.boost.org/LICENSE_1_0.txt

I suggest downloading and putting that into the `ClipperBuilder` repository, then you can just `install_license LICENSE_1_0.txt`.

Reply:
Ok, I updated the sha to include:
https://github.com/SimonDanisch/ClipperBuilder/commit/d9e011161e5f4e161137af19d0f5da6dc5764520